supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-data-grid.adoc

```diff
@@ -0,0 +1,113 @@
+[[red-hat-data-grid-conventions]]
+
+For documentation questions, contact jdg-docs@redhat.com.
+
+[discrete]
+[[cache-manager]]
+==== image:images/yes.png[yes] Cache Manager (noun)
+*Description*: Interface for creating caches and managing cache lifecycles. Always spell as two words with capital letters when referring to the abstract notion of a Cache Manager. When referring to specific interfaces, use `CacheManager`, `EmbeddedCacheManager`, or `RemoteCacheManager`, marked up appropriately.
```

Comment:
[GLOBAL] comment: I suggest to go through and change all the entries into complete sentences.